packages/home-spa/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "webpack-dev-server --open",
-    "build": "webpack --mode production",
+    "build": "webpack",
```

Comment:
Can you add this back? The final build needs to be prod.
As per your webpack config, it should not remove the comments even for prod build.

Reply:
@deshmukhmayur It overwrites the config once the mode is Production

Reply:
Removed the `html-loader`